src/Chapter04/Listing04.22.NestedIf.cs

```diff
@@ -1,45 +1,45 @@
 // Justification: Checking for null isn't discussed yet.
 #pragma warning disable CS8604 // Possible null reference argument
-namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter04.Listing04_21
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter04.Listing04_22
 {
-    #region INCLUDE
     public class TicTacToeTrivia
     {
         public static void Main()
         {
+            #region INCLUDE
             int input;    // Declare a variable to store the input
 
-            System.Console.Write(
+            Console.Write(
                 "What is the maximum number " +
                 "of turns in tic-tac-toe?" +
                 " (Enter 0 to exit.): ");
 
             // int.Parse() converts the ReadLine()
             // return to an int data type
-            input = int.Parse(System.Console.ReadLine());
+            input = int.Parse(Console.ReadLine());
 
             if(input <= 0) // line 16
                 // Input is less than or equal to 0
-                System.Console.WriteLine("Exiting...");
+                Console.WriteLine("Exiting...");
             else
                 if(input < 9) // line 20
-                    // Input is less than 9
-                    System.Console.WriteLine(
+                              // Input is less than 9
+                Console.WriteLine(
```

Comment:
The indentations see odd to me here, Console.WriteLine appears in the same column as the if and the comment way to the right. I think the console and comment should be in the same comment and both 4 spaces in from the if

Reply:
Good catch